copy.sh

```diff
@@ -45,7 +45,7 @@ case $lines in
         printf "$stdin" | $pbcopy
         ;;
     *)
-        filter="$(get_filter "${F:-"fzy:fzf-tmux:fzf:peco:percol:zaw"}")"
+        filter="$(get_filter "${F:-"fzf:fzy:peco:percol:zaw"}")"
```

Comment:
Oh I known. The second commit should be pushed to another branch. Otherwise all commits to the forked repo on master will append to the pull request automatically.